src/coreclr/nativeaot/Runtime/DebugHeader.cpp

```diff
@@ -76,7 +76,7 @@ struct DotNetRuntimeDebugHeader
     // This counter can be incremented to indicate breaking changes
     // This field must be encoded little endian, regardless of the typical endianness of
     // the machine
-    const uint16_t MajorVersion = 1;
+    const uint16_t MajorVersion = 2;
```

Comment:
cc @noahfalk This is revving the debugger contract.

Reply:
There are more changes when native side is updated, but for the most part the observable part is that a few flags have moved